composer.json

```diff
@@ -11,13 +11,18 @@
             "email": "chobieeee@php.net"
         }
     ],
-    "minimum-stability": "dev",
+    "config": {
+        "platform": {
+            "php": "5.3.29"
```

Comment:
Won't this affect library users having newer PHP versions, e.g. by always downloading PHP 5.3 compatible dev dependencies?